.github/labeler-pull-request-triage.yml

```diff
@@ -234,6 +234,9 @@ service/portal:
 service/postgresql:
   - internal/services/postgres/**/*
 
+service/postgresqlhsc:
+  - internal/services/postgresqlhsc/**/*
```

Comment:
as this is now part of cosmos db should it not be part of that service folder?

Reply:
Yes. It's part of cosmos db. So I updated code.